bump_version

```diff
@@ -45,6 +51,23 @@ echo $old_version
 old_version_sed="$( sed 's/\./\\./g' <<<"${old_version}" )" # Escape the '.' chars
 sed -i "s/${old_version_sed}/${new_version}/g" ${build_file}
 
+# update nuget package version
+for f in ${mediabrowser_common} ${jellyfin_data} ${mediabrowser_controller} ${mediabrowser_model} ${emby_naming}
+do
+    code ${f}
```

Comment:
What is this and what does it do

Reply:
This is a leftover from testing the loop. It opened MS VS Code. I'll remove it